assets/js/components/Vehicle.vue

```diff
@@ -55,6 +80,12 @@ export default {
 		removeTargetTime: function () {
 			this.$emit("target-time-removed");
 		},
+		removeVehicle: function () {
+			this.$emit("vehicle-removed");
```

Comment:
Nicht sicher ob ichs richtig verstehe. Mir gefallen die Eventnamen nicht. Hier ist das Fzg ja noch nicht weg, sondern genau das *soll* passieren. Wäre da vehicle-remove nicht der bessere Name? In höheren Schichten würde ich mich sonst immer fragen- ohne die Kenntnis hier- ob das nicht schon erfolgt ist. Das Remove API gibts schon, das vehicles/vehicle select API dauert etwas, ich hadere da noch mit dem Backend.

Reply:
Ja, verstehe ich. Ich bin hier erst mal dem Schema der Target Charge Events gefolgt um in sich konsistent zu sein und nicht zu viel umzubauen. Ich bau die eventnamen dann jetzt generell um.